lib/protocol/SFTP.js

```diff
@@ -7,7 +7,9 @@ const {
   Readable: ReadableStream,
   Writable: WritableStream
 } = require('stream');
-const { inherits, isDate } = require('util');
+const { inherits } = require('util');
+
+const isDate = (arg) => arg instanceof Date
```

Comment:
The following would be better as simply using `instanceof` is not foolproof:

```suggestion
const { inherits, types: { isDate } } = require('util');
```